docs/first_steps/first_steps.rst

```diff
@@ -110,63 +110,88 @@ has additional elements:
 
     Using Lifespan QoS will not have any visual effect.
 
-Other Options
--------------
+Participant configuration
+-------------------------
 
-The eProsima Shapes Demo application allows the user to define additional options.
-To see the Options window, please go to *Options->Preferences* in the main bar.
+The eProsima Shapes Demo application allows the user to define Participant policies.
+To see the Options window, please go to *Options->Participant Configuration* in the main bar.
 The following image shows the Options Menu.
 
-.. image:: /01-figures/options.png
+.. image:: /01-figures/participant.png
    :scale: 75 %
    :alt: Options Window
    :align: center
 
-The user can customize several aspects of Shapes Demo operation:
+- **Transport Protocol:**  You can select between UDP protocol, TCP LAN Server, TCP WAN Server or TCP Client,
+  and Shared Memory protocol.
 
-- **Transport Protocol:** UDP is the default transport protocol for *Fast DDS* but TCP protocol is also available
-  (see
-  `Fast DDS Transports Documentation <https://fast-dds.docs.eprosima.com/en/latest/fastdds/transport/transport.html>`_).
+  - **UDP:** With UDP Protocol the applicacion will work sending multicast packets to communicate with other apps.
+    (See
+    `Fast DDS UDP Transport Documentation <https://fast-dds.docs.eprosima.com/en/v2.3.4/fastdds/transport/udp/udp.html#transport-udp-udp>`_).
```

Comment:
Are we sure we want to leave the link to a specific Fast DDS version? It is true that leaving the `latest` version can eventually broke the link if Fast DDS documentation is refactored, but I think it is a better option than updating all the links with each release.

This applies to several other links in this section.

Reply:
:+1: